### PR TITLE
Feat: Squeezing

### PR DESCRIPTION
**Pull request by efstajas**
Adds the ability to squeeze incoming funds per sender.

Also includes a fix for #284.
Also now shows the real collected amount in any collect operation on the success step.

TODO:
- [x] Open a companion PR in docs repo to update the collect flow docs.
- [x] E2E Test.

### CODE REVIEW

**Comment by efstajas** [comment]
Docs PR: https://github.com/radicle-dev/radicle-drips-docs/pull/10

File: src/lib/flows/collect-flow/collect-amounts.svelte [hunk 1]
```diff
@@ -119,44 +231,88 @@
...
+    >
+  </div>
+  <div class="squeeze-section">
+    <Toggleable label="Include funds from current cycle" bind:toggled={squeezeEnabled}>
+      <p>
+        Select which senders from the current cycle you would like to collect from. The network fee
+        for collecting increases with each selected sender.
+      </p>
+      <AnnotationBox type="warning">
+        The amounts shown below are estimated based on your system time, and the value you actually
```
Comment (evvvritt) [inline src/lib/flows/collect-flow/collect-amounts.svelte:256]:
```suggestion
        The amounts shown below are estimated based on your system time so the value you 
```

**Review by evvvritt — CHANGES_REQUESTED** [review]
nice start!! some things:
(1. discussed tx error when splits exist)
2. awkward animation:
Screen Shot 2023-01-23 at 17 39 22

**Comment by efstajas** [comment]
> 2. awkward animation

Fixed that, waiting for squeezing fix 👍

**Comment by efstajas** [comment]
@evvvritt SDK updated, splitting fixed 👌

**Review by evvvritt — APPROVED** [review]
approved!
one last thing would be improve this success panel on mobile.
maybe "...your wallet address." and link to the etherscan tx? (omitt address)
or css break-all on address string, or use the abbreviated address component
Screen Shot 2023-01-24 at 13 41 28

**Pull request merged**